crates/net/network/src/transactions/mod.rs

```diff
@@ -2125,6 +2125,99 @@ mod tests {
         handle.terminate().await;
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_handle_incoming_transactions_hashes() {
+        reth_tracing::init_test_tracing();
```

Comment:
this test is always quite long, can we add some docs, what this does exactly

Reply:
@mattsse updated!